apps/els_lsp/src/els_code_lens_provider.erl

```diff
@@ -47,7 +47,7 @@ run_lenses_job(Uri) ->
         title => <<"Lenses">>,
         on_complete =>
             fun(Lenses) ->
-                els_provider ! {result, Lenses, self()},
+                els_server ! {result, Lenses, self()},
```

Comment:
I wonder, if it would make sense to have some mechanism and abstraction around this - raw sends always make me nervous

Reply:
Will follow up.

Reply:
https://github.com/erlang-ls/erlang_ls/issues/1334